scripts/publish-devtools.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Usage: node scripts/publish-devtools.js
+ */
+
+const promisify = require(`promisify-node`);
+const fs = promisify(`fs`);
```

Comment:
No biggie but are you actually using fs functions that don't have sync versions? e.g. it might simplify this a bit (and be more standard Node) just to do `fs.writeFileSync(...)` instead of `await fs.writeFile(...)`.

Reply:
Not really, I just made it a habit of using promisified versions of fs in node. In this case it doesn't matter since its a script and we don't care if it's blocking. Moved to *Sync versions